packs/data/feats.db/bloodline-resistance.json

```diff
@@ -29,10 +29,7 @@
                 "predicate": {
                     "any": [
                         "magical",
-                        "arcane",
-                        "divine",
-                        "occult",
-                        "primal"
```

Comment:
You want to keep these. Many effects have one of the tradition traits, but not the magical traits. All effects with a tradition trait inherit the magical trait by default, and would still trigger the bonus.